lib/index.js

```diff
@@ -69,6 +69,9 @@ module.exports = function(options) {
   // function to declare the api versionused for the request
   options.getApiVersion = options.getApiVersion || noop;
 
+  // logBody option
+  options.logBody = options.logBody == false ? false : true;
```

Comment:
Let's simplify that boolean logic.
`options.logBody = (options.logBody !== false);` is functionally equivalent

but why can't we just do:
`options.logBody = Boolean(options.logBody)`

or simply leave it alone since we don't need to affect its truthyness?
If we want this to default to `true` it should be initially set to true.